This patch makes libiris treat JIDs as not being case sensitive


Index: iris/xmpp-core/jid.cpp
===================================================================
--- iris/xmpp-core/jid.cpp	(revision 513088)
+++ iris/xmpp-core/jid.cpp	(working copy)
@@ -252,6 +252,9 @@
 		b = d;
 	else
 		b = n + '@' + d;
+	
+	b=b.toLower(); // JID are not case sensitive
+	
 	if(r.isEmpty())
 		f = b;
 	else
